### DISCUSSION TITLE
[Suggestion] Movement particles

### DISCUSSION THREAD

**Discussion by ImmuSweets** [tsunamayo/Starship-EVO#4662] [Suggestions]
So I've noticed that unless you are traveling at light cruise, there aren't any particle effects, and unless you have a target, or nearby object, you can't really tell you are moving. I think that once you go over 100 m/s there should be at least some minor particles that you seem to "fly by". Non warp, and non light cruise just seems to be missing that sense of motion, especially when you are far from any objects.

**Comment by Gristle0** [comment]
Space dust
